ci/ray_ci/automation/docker_tags_lib.py

```diff
@@ -169,6 +176,88 @@ def delete_tag(tag: str, docker_hub_token: str) -> bool:
     return True
 
 
+def _is_release_tag(
+    tag: str,
+    release_versions: Optional[List[str]] = None,
+) -> bool:
+    """
+    Check if tag is a release tag & is in the list of release versions.
+    Args:
+        tag: Docker tag name
```

Comment:
nit: perhaps document the format of the tag name here

Reply:
done